src/openrct2/world/footpath.c

```diff
@@ -449,7 +449,7 @@ money32 footpath_remove_real(sint32 x, sint32 y, sint32 z, sint32 flags)
 		sub_6A759F();
 	}
 
-	return (flags & (1 << 5)) || (gParkFlags & PARK_FLAGS_NO_MONEY) ? 0 : -MONEY(10,00);
+	return (flags & (1 << 5)) || (gParkFlags & PARK_FLAGS_NO_MONEY) || map_element_is_ghost(mapElement) ? 0 : -MONEY(10,00);
```

Comment:
While we're here, we should also probably replace that (1 << 5) magic number with the appropriate constant, if only so that we can name the variable something sane...